src/components/__tests__/FontAwesomeIcon.test.js

```diff
@@ -1,14 +1,56 @@
 import Vue from 'vue/dist/vue'
 import FontAwesomeIcon from '../FontAwesomeIcon'
 import { library } from '@fortawesome/fontawesome-svg-core'
-import { faCoffee, faCircle } from '../__fixtures__/icons'
-import { compileAndMount, mountFromProps } from '../__fixtures__/helpers'
+import { faXmark, faClose } from '@fortawesome/free-solid-svg-icons'
+import { faCoffee, faCircle, faSpartan } from '../__fixtures__/icons'
+import { coreHasFeature, REFERENCE_ICON_BY_STYLE, compileAndMount, mountFromProps } from '../__fixtures__/helpers'
 
 beforeEach(() => {
-  library.add(faCoffee, faCircle)
+  library.add(faCoffee, faCircle, faSpartan)
   Vue.component('font-awesome-icon', FontAwesomeIcon)
 })
 
+test('using a FAT icon with array format', () => {
+  const vm = mountFromProps({ icon: ['fat', 'spartan'] })
+
+  expect(vm.$el.tagName).toBe('svg')
+  expect(vm.$el.classList.contains('fa-spartan')).toBeTruthy()
+})
+
+if (coreHasFeature(REFERENCE_ICON_BY_STYLE)) {
+  test('find a free-solid-svg-icon with array format', () => {
```

Comment:
@jasonlundien These two alias tests are probably not considered part of the `REFERENCE_ICON_BY_STYLE` feature. What do you say we create a `ICON_ALIASES` feature and segregate these so that it's a little more clear what it is we are testing?

Reply:
ok, understood.  I think this is exactly what you were saying Friday.  Definitely makes sense now !!!